aait/backend/group-2B/Application/DTOs/Authentication/Validators/IUserDtoValidator.cs

```diff
@@ -0,0 +1,25 @@
+using FluentValidation;
+
+namespace SocialSync.Application.DTOs.Authentication.Validators;
+
+public class IUserDtoValidator : AbstractValidator<IUserDto>
```

Comment:
it's worth noting that in C#, interface names typically start with the letter "I" followed by a PascalCase name, whereas class names typically start with a PascalCase name without the "I" prefix.

Reply:
Noted. I wanted it as an interface.